Source/JavaScriptCore/runtime/Symbol.cpp

```diff
@@ -85,7 +85,7 @@ void Symbol::destroy(JSCell* cell)
 
 String Symbol::descriptiveString() const
 {
-    return makeString("Symbol(", String(m_privateName.uid()), ')');
+    return makeString("Symbol("_s, String(m_privateName.uid()), ')');
```

Comment:
Should be able to pass `m_privateName.uid()` without `String` to avoid reference count churn.